$OpenBSD$

- use public domain Korn shell
- add OpenBSD mem size calculation

--- bin/netbeans.orig	Sun Jun 26 15:24:08 2016
+++ bin/netbeans	Sun Jun 26 15:26:10 2016
@@ -156,6 +156,10 @@ heap_size () {
         mem=`cat /proc/meminfo | grep MemTotal | tr -d [:space:][:alpha:]:`
         mem=`expr $mem / 1024`
         ;;
+    OpenBSD*)
+        mem=`/sbin/sysctl hw.usermem | tr -d [:space:][:alpha:]=.:`
+        ((mem=$mem/1048576))
+        ;;
     SunOS*)
         mem=`/usr/sbin/prtconf | grep Memory | /usr/bin/tr -dc '[0-9]'`
         ;;
@@ -209,9 +213,9 @@ launchNbexec() {
     done | head -n 1`
     sh=sh
     # #73162: Ubuntu uses the ancient Bourne shell, which does not implement trap well.
-    if [ -x /bin/bash ]
+    if [ -x /bin/ksh]
     then
-        sh=/bin/bash
+        sh=/bin/ksh
     fi
     if [ "${founduserdir}" = "yes" ]; then
         exec $sh "$nbexec" "$@"
@@ -245,6 +249,7 @@ case "`uname`" in
             --jdkhome '"$netbeans_jdkhome"' \
             --branding nb \
             --clusters '"$netbeans_clusters"' \
+            -J-Dawt.useSystemAAFontSettings=lcd \
             -J-Dnetbeans.importclass=org.netbeans.upgrade.AutoUpgrade \
             -J-Dnetbeans.accept_license_class=org.netbeans.license.AcceptLicense \
             ${netbeans_default_options} \
